docs/releases/upgrading.md

```diff
@@ -1,153 +1,109 @@
+Also, Long Term Support releases will ensure compatibility with at least one [Django Long Term Support release](https://www.djangoproject.com/download/#supported-versions).
 
-For dates of past and upcoming releases and support periods, see `Release Schedule <https://github.com/wagtail/wagtail/wiki/Release-schedule>`_.
+For dates of past and upcoming releases and support periods, see [Release Schedule](https://github.com/wagtail/wagtail/wiki/Release-schedule).
 
-Deprecation policy
-==================
+## Deprecation policy
 
 Sometimes it is necessary for a feature release to deprecate features from previous releases. This will be noted in the "Upgrade considerations" section of the release notes.
 
 When a feature is deprecated, it will continue to work in that feature release and the one after it, but will raise a warning. The feature will then be removed in the subsequent feature release. For example, a feature marked as deprecated in version 1.8 will continue to work in versions 1.8 and 1.9, and be dropped in version 1.10.
 
 
-Upgrade process
-===============
+## Upgrade process
 
 We recommend upgrading one feature release at a time, even if your project is several versions behind the current one. This has a number of advantages over skipping directly to the newest release:
 
-* If anything breaks as a result of the upgrade, you will know which version caused it, and will be able to troubleshoot accordingly;
-* Deprecation warnings shown in the console output will notify you of any code changes you need to make before upgrading to the following version;
-* Some releases make database schema changes that need to be reflected on your project by running ``./manage.py makemigrations`` - this is liable to fail if too many schema changes happen in one go.
+- If anything breaks as a result of the upgrade, you will know which version caused it, and will be able to troubleshoot accordingly;
+- Deprecation warnings shown in the console output will notify you of any code changes you need to make before upgrading to the following version;
+- Some releases make database schema changes that need to be reflected on your project by running `./manage.py makemigrations` - this is liable to fail if too many schema changes happen in one go.
 
 Before upgrading to a new feature release:
 
-* Check your project's console output for any deprecation warnings, and fix them where necessary;
-* Check the new version's release notes, and the :ref:`compatible_django_python_versions` table below, for any dependencies that need upgrading first;
-* Make a backup of your database.
+- Check your project's console output for any deprecation warnings, and fix them where necessary;
+- Check the new version's release notes, and the [Compatible Django / Python versions](compatible_django_python_versions) table below, for any dependencies that need upgrading first;
```

Comment:
```suggestion
- Check the new version's release notes, and the [](compatible_django_python_versions) table below, for any dependencies that need upgrading first;
```

If you want to just inherit the link's label from the linked page you can just write `[]` (with no label).